port_ocean/exceptions/port_defaults.py

```diff
@@ -14,3 +14,13 @@ def __init__(
 
 class UnsupportedDefaultFileType(BaseOceanException):
     pass
+
+
+class DefaultsProvisionFailed(BaseOceanException):
+    def __init__(
+        self,
+        retries,
+    ):
+        super().__init__(
+            f"Failed to retrieve integration config after {retries} attempts"
```

Comment:
does it mean that the integration won't continue to run after this issue happens

Reply:
Yes, and it'll be in the logs